instrumentation/jdbc/jdbc.gradle

```diff
@@ -21,9 +21,6 @@ javacc {
   }
 }
 
-tasks.withType(com.github.sherter.googlejavaformatgradleplugin.VerifyGoogleJavaFormat).configureEach {
```

Comment:
These are generated files and should be excluded from format check.

Reply:
spotless appears to be smarter and excludes files under `build/generated`, so this isn't needed anymore